packages/node_modules/@node-red/nodes/core/common/91-global-config.html

```diff
@@ -0,0 +1,19 @@
+
+<script type="text/html" data-template-name="global-config">
+    <div class="form-tips">
+        <span data-i18n="[html]global-config.tip"></span>
+    </div>
+</script>
+
+<script type="text/javascript">
+    RED.nodes.registerType('global-config',{
+        category: 'config',
+        defaults: {
+            name: { value: "" },
+            env: { value: [] },
+        },
+        credentials: {
+            map: { type: "map" }
+        }
+    });
+</script>
```

Comment:
```suggestion
<script type="text/html" data-template-name="global-config">
    <div class="form-row">
        <button class="red-ui-button" type="button" id="node-input-edit-env-var" data-i18n="editor:env-var.header"></button>
    </div>
</script>

<script type="text/javascript">
    RED.nodes.registerType('global-config',{
        category: 'config',
        defaults: {
            name: { value: "" },
            env: { value: [] },
        },
        credentials: {
            map: { type: "map" }
        },
        oneditprepare: function() {
            $('#node-input-edit-env-var').on('click', function(evt) {
                RED.actions.invoke('core:show-user-settings', 'envvar')
            })
        }
    });
</script>
```

Reply:
Thank you.  It would be beneficial to be able to go directly to the settings as you suggested.
The new UI seems too simple with just a button. How about adding a description like this:
![スクリーンショット 2022-12-06 13 07 44](https://user-images.githubusercontent.com/30289092/205811431-2e8add55-3244-499b-bc90-140917e118bc.png)